tests/system/large/test_streaming.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+import bigframes.streaming
+
+
+def test_streaming_to_bigtable():
+    # launch a continuous query
+    sql = """SELECT
+        body_mass_g, island as rowkey
+        FROM birds.penguins"""
+    query_job = bigframes.streaming.to_bigtable(
+        sql,
```

Comment:
Is there a way that we can check these parameters are kept? If possible may worth adding tests to check.

Reply:
For job_id, yes, added it. For the other parameters, they pass through to bigtable or the client library and aren't accessible on the output QueryJob object, so there isn't much we can do to check them, unfortunately. But fortunately they just pass through, and our documentation points to the parameters they map to, so we wouldn't necessarily need to test that anyway.